tree/ntuple/v7/src/RPageStorage.cxx

```diff
@@ -344,14 +344,16 @@ void ROOT::Experimental::Detail::RPageSink::CommitSealedPage(
 }
 
 
-void ROOT::Experimental::Detail::RPageSink::CommitCluster(ROOT::Experimental::NTupleSize_t nEntries)
+std::uint64_t ROOT::Experimental::Detail::RPageSink::CommitCluster(ROOT::Experimental::NTupleSize_t nEntries)
 {
-   auto locator = CommitClusterImpl(nEntries);
+   auto nbytes = CommitClusterImpl(nEntries);
 
    R__ASSERT((nEntries - fPrevClusterNEntries) < ClusterSize_t(-1));
    fDescriptorBuilder.AddCluster(fLastClusterId, RNTupleVersion(), fPrevClusterNEntries,
                                  ClusterSize_t(nEntries - fPrevClusterNEntries));
-   fDescriptorBuilder.SetClusterLocator(fLastClusterId, locator);
+   // TODO(jblomer): Remove me with the v1 Serialization
+   // For now, we set a non-zero locator to make sure current reading code does not choke
```

Comment:
Do we already have a version number identifier? (if so, adding an assert that it is less than 1 here would help).

Reply:
Yep, [added](https://github.com/root-project/root/pull/8703/commits/15aafca255a9d448efeaf1a8b05913300c14df28)

Reply:
Thanks.